packages/app/src/app/overmind/effects/preview.ts

```diff
@@ -0,0 +1,52 @@
+import { blocker } from 'app/utils/blocker';
```

Comment:
In the future it would be nice if we could incorporate `@codesandbox/executors` in this. This was my goal with that project, that we'd have a single interface to send updates to files to, and the `executors` package would either let the browser know for the client sandboxes or the server for contianer sandboxes. This way we abstract that difference away.